src/Razor/src/Microsoft.VisualStudio.RazorExtension/RazorPackage.cs

```diff
@@ -3,25 +3,25 @@
 
 using System;
 using System.Runtime.InteropServices;
+using System.Threading;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
+using Task = System.Threading.Tasks.Task;
 
 namespace Microsoft.VisualStudio.RazorExtension
 {
-    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [AboutDialogInfo(PackageGuidString, "ASP.NET Core Razor Language Services", "#110", "#112", IconResourceID = "#400")]
     [Guid(PackageGuidString)]
-    public sealed class RazorPackage : Package
+    public sealed class RazorPackage : AsyncPackage
     {
         public const string PackageGuidString = "13b72f58-279e-49e0-a56d-296be02f0805";
 
         private const string CSharpPackageIdString = "13c3bbb4-f18f-4111-9f54-a0fb010d9194";
 
-        protected override void Initialize()
+        protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
```

Comment:
We never use `progress`, should we start or does it not really make sense for our scenario?

Reply:
We may eventually but for now we won't be doing anything time consuming enough to warrant any progress other than 0-100% (represented by this method returning).